modules-sdks/pax-terminal/instore-solution/NET/Methods/requesttodisplayasync.md

```diff
@@ -0,0 +1,21 @@
+---
+title: RequestToDisplayAsync
+description: Task\<NexoRequestResult\> RequestToDisplayAsync(string message)
+---
+
+Call RequestToDispalyAsync to display a message on the terminal. The method may only be called before or after PaymentRequest has been sent to the terminal, which means it may be sent after a GetPaymentInstrument but not during the actual payment which in general starts when the amount is known. 
```

Comment:
```suggestion
Call RequestToDispalyAsync to display a message on the terminal. The method may only be called before or after a PaymentRequest has been sent to the terminal, which means it may be sent after a GetPaymentInstrument but not during the actual payment which in general starts when the amount is known. 
```